Sources/RInAppMessaging/InAppMessagingModule.swift

```diff
@@ -160,7 +160,7 @@ extension InAppMessagingModule {
 
 // MARK: - ErrorDelegate methods
 extension InAppMessagingModule {
-    func didReceiveError(sender: ErrorReportable, error: NSError) {
+    func didReceive(error: NSError) {
```

Comment:
Do we want to force a major version bump with this change to public API?

Reply:
Ah right. I forgot that `ErrorDelegate` is public. I think it's not worth it. I'll revert this change

Reply:
Wait, `RInAppMessagingErrorDelegate` is public, not `ErrorDelegate`. So there's no public API change here